trompace/mutations/controlaction.py

```diff
@@ -46,7 +46,7 @@ def mutation_create_controlaction(name: str, description: str, actionStatus: str
     args = {
```

Comment:
Can you add a check here like we have for languages: https://github.com/trompamusic/trompa-ce-client/blob/8b637389ac9157bc2ca9140e8060c6b4e931dc80/trompace/mutations/person.py#L72
You should add a new Exception.

It'd also be good to update the tests in https://github.com/trompamusic/trompa-ce-client/blob/43ac49c728b5016c360b5b46b13ebaed2157ea23/tests/test_controlaction.py to ensure that we correctly raise this exception.